### ISSUE TITLE
cljr-clean-ns should have an option to disable breaking long lines

### ISSUE THREAD

**Issue by thiagokokada** [clojure-emacs/refactor-nrepl#259]
## Expected behavior

Running `M-x cljr-clean-ns` breaks namespaces lines longer than 80 characters. While this behavior may be desired in some projects, it would be interesting to have an option to disable this behavior.

## Actual behavior

`M-x cljr-clean-ns` always break lines and there doesn't seem to have a way to disable it.

## Steps to reproduce the problem

Run `cljr-clean-ns` with the following file:
```clojure
(ns test
  (:require  [clojure.test :as this.is.a.very.big.clojure.test.alias.bigger.than.eighty.lines]
             [clojure.test :as this.is.another.very.big.clojure.test.alias.bigger.than.eighty.lines]))

(this.is.a.very.big.clojure.test.alias.bigger.than.eighty.lines/testing "blah")
```
This is the result:
```clojure
(ns test
  (:require [clojure.test
             :as
             this.is.a.very.big.clojure.test.alias.bigger.than.eighty.lines]))

(this.is.a.very.big.clojure.test.alias.bigger.than.eighty.lines/testing "blah")
```

This is the desired result:
```clojure
(ns test
  (:require [clojure.test :as this.is.a.very.big.clojure.test.alias.bigger.than.eighty.lines]))

(this.is.a.very.big.clojure.test.alias.bigger.than.eighty.lines/testing "blah")
```
### clj-refactor.el and refactor-nrepl version information

```clj-refactor 2.5.0-SNAPSHOT (package: 20190618.716), refactor-nrepl 2.5.0-SNAPSHOT```

### CIDER version information

```
;; CIDER 0.22.0snapshot (package: 20190619.745), nREPL 0.6.0
;; Clojure 1.10.1, Java 1.8.0_212
```

### Emacs version

```GNU Emacs 26.1 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.24.5) of 2019-06-16```

### Operating system

```
$ cat /etc/os-release
NAME=NixOS
ID=nixos
VERSION="19.03.172921.d77e3bd6613 (Koi)"
VERSION_CODENAME=koi
VERSION_ID="19.03.172921.d77e3bd6613"
PRETTY_NAME="NixOS 19.03.172921.d77e3bd6613 (Koi)"
LOGO="nix-snowflake"
HOME_URL="https://nixos.org/"
SUPPORT_URL="https://nixos.org/nixos/support.html"
BUG_REPORT_URL="https://github.com/NixOS/nixpkgs/issues"
```

**Comment by expez** [comment]
The goal of this op is to:

- Remove unneeded stuff
- Sort so it's easy to scan the ns form to find a particular dep if needed (e.g. during code review on github)

Having a great pretty-printer has turned into a bit of a non-goal because it's surprisingly hard to do that.  Ideally I would like to unload the final pretty-printing step to another project, but none of them perform better than the turn I wrote myself.  I tried all that were available at the time.  Perhaps the situation has changed now.

tl;dr this would've been nice, but I don't think we should spend time on this (much bigger fish to fry!) and I don't want to incur any more complexity to support this in an area of the codebase that is already in a very bad state.

**Comment by jumarko** [comment]
I would really appreciate this feature - would like to use 100 chars limit instead of 80 (or whatever it uses).

**Comment by jumarko** [comment]
Excellent, thanks!
